docs/user-manual/playlists.md

```diff
@@ -37,6 +37,7 @@ Smart blocks are automatically filled with media files from the LibreTime librar
 To create a smart block, click the **Smartblocks** button on the left sidebar, and select **New** from the toolbar. Like a playlist, smart blocks can have a title and description, which you can edit. This helps you find relevant smart blocks in searches.
 
 Fill out the smart block's **Name**, **Search Criteria**, and **Limit to** sections. The search criteria can be any one of LibreTime's metadata categories, such as **Title**, **Creator** or **Genre**. The modifier depends on whether the metadata in question contains letters or numbers. For example, **Title** has modifiers including _contains_ and _starts with_, whereas the modifiers for **BPM** include _is greater than_ and _is in the range_.
+If you want to filter tracks using today's date information, use the `date{}` operator. Format characters are listed in the [php documentation](https://www.php.net/manual/en/datetime.format.php). For example, to filter to tracks with a **Title** that ends in `Instrumental Jan 2024` where `Jan 2024` is the current month and year, you would add a criteria for **Title** with a modifier of **ends with** and a value of `Instrumental date{M Y}`.
```

Comment:
Don't you prefer the name `now` ?

I am wondering if we prefer having templates bracket around the operator ? `{{ date{...} }}` or maybe something more code like: `$date(.*)` ?

Reply:
That is so much better. Changed to `now`, thank you.

As for the syntax:
Hmmm, I'm not sure but there's something about the double curly  with another curly inside, or `$` that is a bit too code like and verbose. I was thinking this is going to be used by people who aren't as "codey" so I was trying to keep it as simple as possible.